tests/dummy/config/environment.js

```diff
@@ -44,6 +44,10 @@ module.exports = function (environment) {
   if (environment === 'production') {
     ENV.locationType = 'hash'
     ENV.rootURL = '/ember-frost-table'
+    ENV.mirageNamespace = 'https://ciena-frost.github.io'
+    ENV['ember-cli-mirage'] = {
+      enabled: true
+    }
```

Comment:
Thanks for fixing this, I wasn't thinking about the `gh-pages` when I turned on mirage in my other PR